Test Failing for BIG-IP platform in Upstream Monitor

### Environment
* Python version:  3.8
* Nautobot version:  ltm-1.6
* nautobot-ssot version:  ltm-1.6

<!-- What did you expect to happen? -->
### Expected Behavior
CI tests to pass.

<!-- What happened instead? -->
### Observed Behavior
See tests failing:

```
System check identified no issues (0 silenced).
......WARNING: All log messages before absl::InitializeLog() is called are written to STDERR
I0000 00:00:1721881114.640414       1 config.cc:230] gRPC experiments enabled: call_status_override_on_cancellation, event_engine_dns, event_engine_listener, http2_stats_fix, monitoring_experiment, pick_first_new, trace_record_callops, work_serializer_clears_time_cache
............................................................................................................F
======================================================================
FAIL: test_get_netmiko_platform_5_bigip (nautobot_ssot.tests.device42.unit.test_utils_device42.TestUtilsDevice42)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/usr/local/lib/python3.8/site-packages/parameterized/parameterized.py", line 533, in standalone_func
    return func(*(a + p.args), **p.kwargs)
  File "/source/nautobot_ssot/tests/device42/unit/test_utils_device42.py", line 175, in test_get_netmiko_platform
    self.assertEqual(device42.get_netmiko_platform(sent), received)
AssertionError: 'bigip_f5' != 'f5_tmsh'
- bigip_f5
+ f5_tmsh
```
<!--
    Describe in detail the exact steps that someone else can take to reproduce
    this bug using the current release.
-->
### Steps to Reproduce
1. Run CI tests against ltm-1.6 branch.
2.
3.

--- a/nautobot_ssot/tests/device42/unit/test_utils_device42.py
+++ b/nautobot_ssot/tests/device42/unit/test_utils_device42.py
@@ -165,7 +165,6 @@ class TestUtilsDevice42(TestCase):
         ("iosxe", "iosxe", "cisco_ios"),
         ("iosxr", "iosxr", "cisco_xr"),
         ("nxos", "nxos", "cisco_nxos"),
-        ("bigip", "f5", "f5_tmsh"),
         ("junos", "junos", "juniper_junos"),
         ("dell", "dell", "dell"),
     ]

